src/components/AddLayerModal/index.tsx

```diff
@@ -158,6 +172,8 @@ export const AddLayerModal: React.FC<AddLayerModalProps> = ({
     closeModal();
   };
 
+  console.log(validationStatus)
```

Comment:
```suggestion
```

Reply:
Sorry, that somehow sneaked it way into the MR.